src/sentry/models/apitoken.py

```diff
@@ -207,38 +207,34 @@ def plaintext_refresh_token(self) -> str:
         return token
 
     def save(self, *args: Any, **kwargs: Any) -> None:
-        if options.get("apitoken.save-hash-on-create"):
-            self.hashed_token = hashlib.sha256(self.token.encode()).hexdigest()
+        self.hashed_token = hashlib.sha256(self.token.encode()).hexdigest()
 
-            if self.refresh_token:
-                self.hashed_refresh_token = hashlib.sha256(self.refresh_token.encode()).hexdigest()
-            else:
-                # The backup tests create a token with a refresh_token and then clear it out.
-                # So if the refresh_token is None, wipe out any hashed value that may exist too.
-                # https://github.com/getsentry/sentry/blob/1fc699564e79c62bff6cc3c168a49bfceadcac52/tests/sentry/backup/test_imports.py#L1306
-                self.hashed_refresh_token = None
+        if self.refresh_token:
+            self.hashed_refresh_token = hashlib.sha256(self.refresh_token.encode()).hexdigest()
+        else:
+            # The backup tests create a token with a refresh_token and then clear it out.
+            # So if the refresh_token is None, wipe out any hashed value that may exist too.
+            # https://github.com/getsentry/sentry/blob/1fc699564e79c62bff6cc3c168a49bfceadcac52/tests/sentry/backup/test_imports.py#L1306
+            self.hashed_refresh_token = None
 
-        if options.get("apitoken.auto-add-last-chars"):
-            token_last_characters = self.token[-4:]
-            self.token_last_characters = token_last_characters
+        token_last_characters = self.token[-4:]
```

Comment:
What is the significance of 4?

Reply:
This grabs the last four characters of the token and sets them for the `token_last_characters` column. 
They are then used in the UI to help users identify their tokens. 

![image](https://github.com/user-attachments/assets/2d102dfd-d5e0-4624-a97d-c1046b049c90)